WalletWasabi.Fluent/ViewModels/AddWallet/AddedWalletPageViewModel.cs

```diff
@@ -18,7 +18,7 @@ private AddedWalletPageViewModel(IWalletSettingsModel walletSettings, WalletCrea
 	{
 		_walletSettings = walletSettings;
 
-		WalletName = walletSettings.WalletName;
+		WalletName = options.WalletName;
```

Comment:
null warnings

Reply:
Done. 
I noticed something, though: It looks suspicious that the wallet name could be null. Is there any scenario in which this can happen?